src/Components/Dashboard/CardDashboard.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./Dashboard.module.css";
+import { useHistory } from "react-router";
+
+const CardDashboard = ({ title, icon, link }) => {
+  const history = useHistory();
+
+  const linked = () => {
+    history.push(`${link}`);
```

Comment:
No es necesario interpolar variables si es lo único que tenes!

```suggestion
    history.push(link);
```